llpc/context/llpcShaderCache.cpp

```diff
@@ -365,15 +365,16 @@ void ShaderCache::resetCacheFile() {
   m_onDiskFile.close();
   Result fileResult = m_onDiskFile.open(m_fileFullPath, (FileAccessRead | FileAccessWrite | FileAccessBinary));
   assert(fileResult == Result::Success);
-  (void(fileResult)); // unused
 
   ShaderCacheSerializedHeader header = {};
   header.headerSize = sizeof(ShaderCacheSerializedHeader);
   header.shaderCount = 0;
   header.shaderDataEnd = header.headerSize;
   getBuildTime(&header.buildId);
 
-  m_onDiskFile.write(&header, header.headerSize);
+  fileResult = m_onDiskFile.write(&header, header.headerSize);
+  assert(fileResult == Result::Success);
+  (void(fileResult)); // unused
```

Comment:
Can we stick to the same style of unused variables everywhere? I think the canonical one is `(void) x;`

```
➜ /ssd/opensource/llvm/llvm-project git:(main) ✗ 
$ ag '^\s*\(void\)\s*[a-zA-Z]*;' | wc -l
1723
➜ /ssd/opensource/llvm/llvm-project git:(main) ✗ 
$ ag '^\s*\(void\(' | wc -l 
5
```